composer.json

```diff
@@ -48,34 +48,32 @@
         }
```

Comment:
Haven't checked these versions but I am assuming they match the versions of the modules that were installed before.